docs/release-notes/tmux_packaging_log.rst

```diff
@@ -0,0 +1,9 @@
+:Type: TUI
+:Summary: Add packaging log to ``tmux`` tabs
+
+:Description:
+    We already have other important logs in the pre-defined tabs in tmux attached on TTY1.
+    Add the last one missing to easily get access to information about the package installation.
```

Comment:
I don't think this text describes the features. Also, this is not the last one missing.